packages/flutter_tools/test/commands.shard/hermetic/build_web_test.dart

```diff
@@ -60,6 +60,7 @@ void main() {
       null,
       true,
       true,
+      null
```

Comment:
nit: trailing comma:
```suggestion
      null,
```